tests/js/spec/components/globalSdkUpdateAlert.test.tsx

```diff
@@ -0,0 +1,278 @@
+import moment from 'moment';
+
+import {mountWithTheme, screen, userEvent} from 'sentry-test/reactTestingLibrary';
+
+import {InnerGlobalSdkUpdateAlert} from 'sentry/components/globalSdkUpdateAlert';
+import {ALL_ACCESS_PROJECTS} from 'sentry/constants/pageFilters';
+import {PageFilters, ProjectSdkUpdates} from 'sentry/types';
+import {DEFAULT_SNOOZE_PROMPT_DAYS} from 'sentry/utils/promptIsDismissed';
+import useApi from 'sentry/utils/useApi';
+import {OrganizationContext} from 'sentry/views/organizationContext';
+
+jest.mock('sentry/utils/useApi');
```

Comment:
Do we need to mock useApi, the API client is already mocked using the `__mocks__` approach.

Reply:
When using `MockApiClient.addMockResponse` the return value is the jest mock for that API call. This lets you do assertions on it.

```typescript
const updateMock = MockApiClient.addMockResponse({
   url: `/organizations/${orgSlug}/prompts-activity/`,
   method: 'PUT',
   body: {},
   statusCode: 202
});
userEvent.click(await screen.findByText(/Remind me later/));
expect(updateMock).toHaveBeenCalledWith({
  urlExpectation,
  expect.objectContaining({
    data: {feature: 'sdk_updates', ...}
  })
});
```

You shouldn't need the specific API instance as the Jest mock in `app/__mocks__/api.tsx` overwrites all API clients in the test run.